test/integration/registry.go

```diff
@@ -14,9 +14,23 @@
 
 package integration
 
-import "github.com/onosproject/onos-config/test/runner"
+import (
+	"github.com/onosproject/onos-config/test/runner"
+)
 
 var (
 	// Registry holds the registered test names
 	Registry = runner.NewRegistry()
+
+	//AllTests is an example of a test suite
+	AllTests = runner.NewTestSuite("alltests")
+	//SomeTests is an example of a test suite
+	SomeTests = runner.NewTestSuite("sometests")
```

Comment:
also we are not adding any test to these test suites, am I correct ?

Reply:
correct